### PR TITLE
Fix issue #574.

### PR DESCRIPTION
**Pull request by wuestholz**

### CODE REVIEW

File: src/Utilities/PlatformManifest.cs [hunk 1]
```diff
@@ -145,33 +145,33 @@ private void LoadManifestFile()
...
+                            {
+                                DependentPlatforms.Add(new DependentPlatform(childElement.GetAttribute(Attributes.Name), childElement.GetAttribute(Attributes.Version)));
+                            }
                         }
-                        else if (String.Equals(childElement.Name, Elements.DependentPlatform, StringComparison.Ordinal))
-                        {
-                            DependentPlatforms.Add(new DependentPlatform(childElement.GetAttribute(Attributes.Name), childElement.GetAttribute(Attributes.Version)));
-                        }
-                    }
+                    }
```
Comment (rainersigwald) [inline src/Utilities/PlatformManifest.cs]: Nit: trailing whitespace here and on line 164.

**Comment by rainersigwald** [comment]
Looks good to me. Thank you for the contribution!

**Comment by wuestholz** [comment]
@rainersigwald Sorry about that. I fixed the trailing whitespace.

**Comment by wuestholz** [comment]
@rainersigwald Is there anything else I should do before this can be merged?

**Comment by rainersigwald** [comment]
@wuestholz This looks like it's in good shape to me, but we try to follow a two-committers approval process.

@cdmihai, @AndyGerlicher ?

**Comment by radical** [comment]
It would be nice to have a more descriptive commit message though :)

**Comment by wuestholz** [comment]
@rainersigwald I see. I wasn't aware of that. :) Thanks!

@rainersigwald @radical @cdmihai @AndyGerlicher I have just squashed the two commits and made the commit message more descriptive.

**Comment by radical** [comment]
@wuestholz How about:

```
Fix possible NRE in PlatformManifest.LoadManifestFile

Ensure rootElement is not null before accessing it's childNodes
```

I had to carefully look at the patch to see what had really changed. It might be only a "me" problem, but making it clear in the commit message is useful, IMHO We want to reduce the pain for future developers looking at this :)

**Comment by AndyGerlicher** [comment]
I agree @radical , until I looked at this in a different visualizer it looked a lot more complicated than it really is. Change LGTM.

Can you update the commit message for what @radical suggested? Then we'll merge. Thanks for contributing this!

**Comment by wuestholz** [comment]
@AndyGerlicher @radical @rainersigwald I just pushed the changes. Thank you!

**Comment by AndyGerlicher** [comment]
@dotnet-bot test this please

**Pull request merged**